tiatoolbox/cli/tile_server.py

```diff
@@ -0,0 +1,17 @@
+"""Command line interface for TileServer"""
+import click
+
+from tiatoolbox.cli.common import (
+    tiatoolbox_cli,
+)
+
+
+@tiatoolbox_cli.command()
+@click.argument("img_paths", nargs=-1, type=click.Path(exists=True))
+def tile_server(img_paths):  # pragma: no cover
+    """show a slide together with a whole slide overlay"""
+    from tiatoolbox.visualization.tileserver import TileServer
+
+    app = TileServer("TileServer", list(img_paths))
```

Comment:
Can we not run the server in a directory and open image directly from there?

Reply:
do you mean just give a directory and try to open all images in there as layers?

Reply:
We can open individual images here, but yes we should be able to open a directory in future.

Reply:
I am resolving it for the sake of this PR.